From: Kevin Murray <spam@kdmurray.id.au>
Date: Tue, 1 Dec 2015 14:52:40 +1100
Subject: Build lib by default

---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,9 @@
 
 .PHONY: default all zstdprogram clean install uninstall travis-install test clangtest gpptest armtest usan asan uasan
 
-default: zstdprogram
+default:
+	$(MAKE) -C $(ZSTDDIR) $@
+	$(MAKE) -C $(PRGDIR) $@
 
 all: 
 	$(MAKE) -C $(ZSTDDIR) $@
