From: Kevin Murray <spam@kdmurray.id.au>
Date: Tue, 22 Dec 2015 12:31:33 +1100
Subject: Append to CPPFLAGS, don't overwrite

---
 lib/Makefile      | 4 ++--
 programs/Makefile | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/lib/Makefile
+++ b/lib/Makefile
@@ -43,8 +43,8 @@
 
 DESTDIR?=
 PREFIX ?= /usr
-CPPFLAGS= -I.
-CFLAGS ?= -O3
+CPPFLAGS += -I.
+CFLAGS += -O3
 CFLAGS += -std=c99 -Wall -Wextra -Wundef -Wshadow -Wcast-qual -Wcast-align -Wstrict-prototypes -Wstrict-aliasing=1
 FLAGS   = $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) $(MOREFLAGS)
 
--- a/programs/Makefile
+++ b/programs/Makefile
@@ -44,8 +44,8 @@
 
 DESTDIR?=
 PREFIX ?= /usr
-CPPFLAGS= -I../lib -DZSTD_VERSION=\"$(VERSION)\"
-CFLAGS ?= -O3  # -falign-loops=32   # not always beneficial
+CPPFLAGS += -I../lib -DZSTD_VERSION=\"$(VERSION)\"
+CFLAGS  += -O3  # -falign-loops=32   # not always beneficial
 CFLAGS += -std=c99 -Wall -Wextra -Wcast-qual -Wcast-align -Wshadow -Wstrict-aliasing=1 -Wswitch-enum -Wstrict-prototypes -Wundef
 FLAGS   = $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) $(MOREFLAGS)
 
